src/index.js

```diff
@@ -34,44 +35,66 @@ let player = {
 
 let inventory = [];
 
+const TOWN_COMMANDS = {
+  inventory: () => checkInv(inventory),
+
+  greet,
+
+  hunt: () => {
+    enemy = hunt();
+    return enemy.msgs.encounter;
+  },
+};
+
+const FIGHT_COMMANDS = {
+  attack: () => {
+    const response = attack(player, enemy, inventory);
+    if (enemy.hp <= 0) {
+      enemy = null;
+    }
+    return response;
+  },
+
+  check: () => check(enemy),
+};
```

Comment:
You did some backend stuff with Express right? We're going to slowly move towards building something similar to Express (yes *building* Express) for message handlers. It'll be fun I swear. Express is not that complicated on the inside.

More accurately, it'll be more like [Koa](https://koajs.com/), which is sort of the successor to Express, if you are interested in looking at that. Koa is very similar but easier to use than Express, so if you were going to do another web server I'd recommend it.

Reply:
Related to this, I've assigned you to the new issue #14 which is step one of doing this ^

Reply:
Yes, we worked a lot with Express. Koa is new to me and I'll have a look into it! Looking into #14 now.